src/features/HomePage/MoviesGallery/MoviesGallery.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { propTypesShapes } from '../../../constants';
+import MovieCard from './MovieCard/MovieCard';
+import Spinner from '../../../components/Spinner/Spinner';
+import { fetchMovies } from '../../../redux/movies/movies.async.actions';
+import { filteredMoviesSelector, isMoviesLoadingSelector } from '../../../redux/movies/movies.selectors';
+import styles from './MoviesGallery.module.scss';
+
+class MoviesGallery extends Component{
+  componentDidMount = () => {
+    const { fetchMovies } = this.props;
+
+    fetchMovies()
+  };
```

Comment:
1. Why arrow func ?
2. Why you destruct fetchMovies, instead using it directly ? `this.props.fetchMovies()`